torch/nn/utils/stateless.py

```diff
@@ -47,8 +45,10 @@ def _untie_named_tensors_map(
     all_named_tensors.update(module.named_buffers(remove_duplicate=False))
 
     # A map of {tensor: set(all_tied_names)} for all tensor names in the module.
-    tensor_to_tied_names_map: Dict[Tensor, Set[str]] = defaultdict(set)
+    tensor_to_tied_names_map: Dict[Tensor, Set[str]] = {}
```

Comment:
Can we open issues to add support for this in Dynamo?

Reply:
Done! https://github.com/pytorch/pytorch/issues/130557